### ISSUE TITLE
Error when displaying video playback screen with androidUseMediaKit set to true.

### ISSUE THREAD

**Issue by bestriser** [zezo357/flutter_meedu_videoplayer#62]
The following error occurred when displaying the video playback screen with androidUseMediaKit set to true.

When this error occurs, the video cannot be played.

Is there anything I can do?

### pubspec.yaml
```yaml
  flutter_meedu_videoplayer: ^4.2.3
  media_kit_libs_ios_video: ^1.0.4
  media_kit_libs_android_video: ^1.0.3
```

### main.dart
```dart
void main() async {
  WidgetsFlutterBinding.ensureInitialized();
  await initMeeduPlayer(iosUseMediaKit: true, androidUseMediaKit: true);
  );
}
```

### Error & Stack trace
```
Error:
Null check operator used on a null value

Stack trace:
#0      VideoPlayerMediaKit.initStreams.<anonymous closure> (package:video_player_media_kit/video_player_media_kit_platform/video_player_media_kit_platform_ffi.dart:146:58)
#1      _rootRunUnary (dart:async/zone.dart:1406:47)
#2      _CustomZone.runUnary (dart:async/zone.dart:1307:19)
#3      _CustomZone.runUnaryGuarded (dart:async/zone.dart:1216:7)
#4      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:339:11)
#5      _DelayedData.perform (dart:async/stream_impl.dart:515:14)
#6      _PendingEvents.handleNext (dart:async/stream_impl.dart:620:11)
#7      _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:591:7)
#8      _rootRun (dart:async/zone.dart:1390:47)
#9      _CustomZone.run (dart:async/zone.dart:1300:19)
#10     _CustomZone.runGuarded (dart:async/zone.dart:1208:7)
#11     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1248:23)
#12     _rootRun (dart:async/zone.dart:1398:13)
#13     _CustomZone.run (dart:async/zone.dart:1300:19)
#14     _CustomZone.runGuarded (dart:async/zone.dart:1208:7)
#15     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1248:23)
#16     _microtaskLoop (dart:async/schedule_microtask.dart:40:21)
#17     _startMicrotaskLoop (dart:async/schedule_microtask.dart:49:5)
```

### Flutter Doctor
```
[✓] Flutter (Channel stable, 3.7.12, on macOS 13.3.1 22E261 darwin-arm64, locale ja-JP)
    • Flutter version 3.7.12 on channel stable at /Users/yuyayamamura/fvm/versions/stable
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4d9e56e694 (2 weeks ago), 2023-04-17 21:47:46 -0400
    • Engine revision 1a65d409c7
    • Dart version 2.19.6
    • DevTools version 2.20.1

[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
    • Android SDK at /Users/yuyayamamura/Library/Android/sdk
    • Platform android-33, build-tools 33.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14E222b
    • CocoaPods version 1.12.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.77.3)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.62.0

[✓] Connected device (4 available)
    • SH RM12 (mobile)            • 357983102288699           • android-arm64  • Android 11 (API 30)
    • bestriserのiPhoneXs (mobile) • 00008020-000325200A88002E • ios            • iOS 16.4.1 20E252
    • macOS (desktop)             • macos                     • darwin-arm64   • macOS 13.3.1 22E261 darwin-arm64
    • Chrome (web)                • chrome                    • web-javascript • Google Chrome 112.0.5615.137

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Comment by zezo357** [comment]
Thank you for the detailed issue, will test and let you know



Note: I may take some time since I am not free right now


For the meantime you can use it with media kit as false for Android and ios

**Comment by zezo357** [comment]
fixed in `flutter_meedu_videoplayer: ^4.2.4` can you test and let me know?

i recommend not using media_kit for android for now (noticed some problems buffering not showing and pauses)

but its fixed

**Comment by bestriser** [comment]
@zezo357 
Correction, thank you!!
I can play videos on Android MediaKit, After updating to `flutter_meedu_videoplayer: ^4.2.5`!!
